variables.tf

```diff
@@ -150,19 +150,19 @@ variable "ec2_sg_egress_rules" {
 variable "tags" {
   type        = any
   default     = null
-  description = "all tags for all recursives"
+  description = "all tags for all recursively"
```

Comment:
change this description, as it is it says nothing. Something like:

`Sets custom tags. Applies to all resources created within the module.`

Additionally, include and example within the description, since it is an `any` type the format is unclear.